tests/test_fluxIntegration.py

```diff
@@ -6,8 +6,8 @@
 import os
 import sys
 sys.path.insert(0,'..') #This adds the ability to pull Flux profiles from the main folder
-import TACOCAT
+from TACOCAT import Tbulk
 
-Test = TACOCAT.test(2,2)
+x = Tbulk
```

Comment:
Can you explain this unit test?

Does it test each profile?

Reply:
This test isn't needed for this pull request it was left over from when I started creating an integration test for flux but realized I needed to do a pull request for updating TACOCAT to be able to use user selected flux profiles. I deleted the file from this pull request to eliminate further confusion.